x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPolicyRunner.java

```diff
@@ -226,10 +228,7 @@ private XContentBuilder resolveEnrichMapping(final EnrichPolicy policy) {
     private void prepareAndCreateEnrichIndex() {
         long nowTimestamp = nowSupplier.getAsLong();
         String enrichIndexName = EnrichPolicy.getBaseName(policyName) + "-" + nowTimestamp;
-        Settings enrichIndexSettings = Settings.builder()
-            .put("index.auto_expand_replicas", "0-all")
```

Comment:
Initially we should set number of replicas to 0, so that later on we can enable auto expand replicas